charts/stable/plex/SCALE/questions.yaml

```diff
@@ -220,7 +220,7 @@ questions:
                             label: "Port Type"
                             schema:
                               type: string
-                              default: "HTTPS"
+                              default: "HTTP"
```

Comment:
It should work fine with HTTPS, doesn't it?

Reply:
I tested yesterday, and it does not.
At least by hitting portal button.
What I didn't check is if the portal redirected to the correct address, which I'll check when I get home.

Reply:
It needs `HTTP`, it's also confirmed by probes now, with `HTTPS` I get `Startup probe failed: Get "https://172.16.0.37:32400/": EOF`

Reply:
The CI doesn't complain about it though...

Reply:
Tried again now, setting to HTTPS, startup probe fails. setting the same app back to http works. setting same app back to https fails again.
So, nothing  from plex side delaying the startup and making probe fail..

Does CI test it with nodeport?

Reply:
Nodeport has nothing to do with probes.

Reply:
Fuck, I think I know the issue... brb.

Reply:
Try setting path to:
/web/index.html

Reply:
Wait,. try:
`/identity` instead...